lib/code_corps/stripe_service/stripe_file_upload.ex

```diff
@@ -0,0 +1,15 @@
+defmodule CodeCorps.StripeService.StripeFileUploadService do
+  alias CodeCorps.StripeService.Adapters.StripeFileUploadAdapter
+
+  @api Application.get_env(:code_corps, :stripe)
+
+  def create(%{"file" => file, "purpose" => purpose, "stripe_connect_account_id" => stripe_connect_account_id} = attributes) do
+    with {:ok, %Stripe.FileUpload{} = stripe_file_upload} <- @api.FileUpload.create(file, purpose, stripe_connect_account_id),
```

Comment:
@snewcomer I think this is right. `@api` is just `Stripe` from `stripity_stripe`, so you're using the create you made.